app/components/NavUser.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Nav, Button } from 'react-bootstrap';
+import { connect } from 'react-redux';
+
+const NavUser = ({user})=> {
```

Comment:
is there a possibility this could be the UserNav and GuestNav could be one component? and the nav link displayed would be different based on the userType using a ternary? You def use a ternary in the navigation component to do this, but I personally prefer components closer to the root to do less work and the outer ones to take care of heavy lifting. Just a thought!

Reply:
Yup you definitely have a good point to make it into one component, will definitely update with the userType as well